emails/migrations/0037_domain_address_min_length_validator_and_unique_together_user_and_address.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.24 on 2021-12-20 21:11
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emails', '0036_profile_add_index_on_last_account_flagged'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='domainaddress',
+            name='address',
+            field=models.CharField(max_length=64, validators=[django.core.validators.MinLengthValidator(limit_value=1)]),
+        ),
+        migrations.AlterUniqueTogether(
+            name='domainaddress',
+            unique_together={('user', 'address')},
+        ),
+    ]
```

Comment:
There's also already `0037_reply_add_index_on_created_at.py` now too. So this one needs to be `0038_domain_address_min_length_validator_and_unique_together_user_and_address.py`, and then the next one needs to be `0039_reply_add_index_on_created_at.py`